types.go

```diff
@@ -78,6 +78,7 @@ type Store struct {
 	data              *Dict
 	defaultExpiration time.Duration
 	expirationChanged bool
+	expirationLock    sync.RWMutex
```

Comment:
Why do you duplicate this??
Please, use `lock`

Reply:
Alright, sorry.

Reply:
Done